docs/docs-beta/CONTRIBUTING.md

```diff
@@ -97,9 +97,9 @@ Before:
 
 After:
 
-| Key | Value |
-|---|---|
-| `DAGSTER_CLOUD_DEPLOYMENT_NAME` | The name of the Dagster+ deployment. <br/><br/>  **Example:** `prod`. |
+| Key                                  | Value                                                                                                     |
```

Comment:
All formatting not related to `docs/docs-beta/docs/dagster-plus/deployment/management/settings/` is from `yarn lint`